spec/rdf2marc/rdf2model/mappers/leader_spec.rb

```diff
@@ -17,23 +17,47 @@
   end
 
   describe 'record status' do
-    let(:ttl) do
-      <<~TTL
-                          <#{admin_metadata_term}> <http://id.loc.gov/ontologies/bibframe/status> _:b17.
-        _:b17 a <http://id.loc.gov/ontologies/bibframe/Status>;
+    context 'when a literal' do
+      let(:ttl) do
+        <<~TTL
+          <#{admin_metadata_term}> <http://id.loc.gov/ontologies/bibframe/status> _:b17.
+          _:b17 a <http://id.loc.gov/ontologies/bibframe/Status>;
             <http://id.loc.gov/ontologies/bibframe/code> "a"@eng.
-      TTL
+        TTL
+      end
+
+      let(:model) do
+        {
+          bibliographic_level: 'item',
+          type: 'language_material'
+          # NOTE: It is not mapped; it is defaulted in the Leader struct
+          # record_status: 'a'
```

Comment:
@justinlittman This is a change of behavior, so I thought I'd flag it in the test suite. If you'd prefer we remove these tests entirely, that's also fine by me.